id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md

```diff
@@ -0,0 +1,25 @@
+# id_splitter_nn
+
+Model that contains source code identifier splitter BiLSTM weights.
+
+Example:
+
+```python
+from sourced.ml.models.id_splitter import IdentifierSplitterNN
+id_splitter = IdentifierSplitterNN().load("522bdd11-d1fa-49dd-9e51-87c529283418")
```

Comment:
Why didn't you add the backend argument in the example?

Reply:
because I completely forgot, mostly.
duh, sorry

Reply:
Can this be edited ?

Reply:
I'm not sure, the thing is that the meta is saved to the `index.json` automatically. Probably if you just edit it in both places it's ok, idk. I'd force-publish it again myself, but idk what's the right way here.

Reply:
@irinakhismatullina fixed

Reply:
Awesome, thank you!